charts/nginx-ingress-controller/test/default.yaml.out

```diff
@@ -563,39 +568,10 @@ webhooks:
       - v1
     clientConfig:
       service:
-        namespace: "default"
         name: nginx-ingress-controller-admission
+        namespace: default
```

Comment:
Is `default` expected here? 👀

Reply:
Yes. Originally the template in the Helm chart had no namespace field at all and relied on Helm to auto-inject it via its `--namespace ...` flag. However in a later release of the nginx chart, they included an explicit namespace field, and since our test.sh renders without `--namespace`, it defaults to "default".

Reply:
I didn't notice this is a test, all good then.